parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -445,6 +445,16 @@ EOF
         echo $(date),$(hostname), "End configuring GPU drivers"
     fi
 
+    export -f enable_managed_gpu_experience
```

Comment:
any reason why we ned an explicit export here ?

Reply:
Without the `export -f ...` the subprocess we create with `bash -cx` won't know about the `enable_managed_gpu_experience` function.